west.yml

```diff
@@ -109,7 +109,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 2dee988030c1048cafea2c06f5fa504322c3afd3
+      revision: pull/774/head 
```

Comment:
Commit message says:

```
-Update the nrfxlib with a fix for
 the PSA cipher_update API.
```

The leading `-` is not part of the expected format. Instead, use something like:

```
Update nrfxlib to bring in a fix for the PSA cipher_update API.
```

Reply:
Removed the - and updated the description here.